testbed/tests/window/test_window.py

```diff
@@ -514,46 +842,183 @@ async def test_move_and_resize(second_window, second_window_probe):
             )
         ],
     )
-    async def test_full_screen(second_window, second_window_probe):
-        """Window can be made full screen"""
-        assert not second_window_probe.is_full_screen
+    async def test_window_state_minimized(second_window, second_window_probe):
+        """Window can have minimized window state."""
```

Comment:
Why do we need explicit tests of minimized state when we've already validated every possible transition? 

Or, if the issue is that there are properties of the final minimized state that need to be validated (which seems possible) - then we should have a standalone test case for each *final* state, parameterised by *original* state, rather than a single full x->y matrix test.

Reply:
I agree with you. No dimensions of any flex widget changes(like it does when it enters MAXIMIZED, FULLSCREEN, or PRESENTATION modes). So, I will remove this test.